user-guide/toc.yml

```diff
@@ -2378,10 +2378,22 @@
               topicUid: UD_APIs_Define_New_API
             - name: Using existing scripts
               topicUid: UD_APIs_Using_existing_scripts
+            - name: Creating APIs and tokens in code
+              topicUid: UD_APIs_Creating_in_code
+            - name: Objects
```

Comment:
S: add an overview page of both items. Now "Defining an new API" is put under objects.
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/115180741/235095991-3d9cea6e-08a6-4a8e-b97c-60c2804d5f12.png)

Reply:
Adding an overview page seemed to have resolved this bug. Thanks.